conda-recipe/meta.yaml

```diff
@@ -120,7 +120,7 @@ outputs:
         - pytest >=3,<4
         - pytest-qt
         # need to help mamba here a bit
```

Comment:
I guess this comment can go now?

Reply:
ah crap I moved stuff around - this should actually stay, but need to move down the `ilastik-pytorch-version-helper-cpu` thing...

Reply:
or the comment up :)

Reply:
ah, nah, this should actually be in test reqs - good catch thx :)